.ameba.yml

```diff
@@ -0,0 +1,2 @@
+Excluded:
+  - ./lib
```

Comment:
Is this necessary since the `/lib/` folder is in the `.gitignore` anyways?

I've never had issues with Ameba raising errors for installed shard dependencies.

Reply:
I had to specifically add this so that ameba didn't report errors for [`taylorfinnell/awscr-signer`](https://github.com/taylorfinnell/awscr-signer).